platform-tests/platform-tests.gradle.kts

```diff
@@ -63,7 +63,7 @@ tasks {
 		useJUnitPlatform {
 			excludeTags("exclude")
 		}
-		jvmArgs("-Xmx1g")
+		jvmArgs("-Xmx1g", "--enable-preview")
```

Comment:
It will no longer be in preview so this can be removed, right?

Reply:
Right. I'll remove `, "--enable-preview"` from that line.